assets/js/src/search.js

```diff
@@ -20,6 +20,7 @@ async function initializeIndex() {
     ],
     minMatchCharLength: 2,
     threshold: 0.2,
+    useExtendedSearch: true,
```

Comment:
I think extended search is for enabling operators: https://www.fusejs.io/examples.html#extended-search

The fuzzy search is controlled by threshold, I believe

Reply:
Thanks! I misinterpreted the doc as extended search enabled all the stuff in the table below.
It does seem to work for the specific case of searching for `create org`. I'll play with the threshold to see if it makes a difference.

Reply:
- With UseExtendedSearch, and bumping the threshold up to 0.3, `create org` finds the topic and no other topics.
- With UseExtendedSearch, and leaving the threshold at 0.2, `create org` finds the topic and no other topics.
- With no UseExtendedSearch, and bumping the threshold up to 0.3, `create org` finds the topic plus 12 other topics. 
- With no UseExtendedSearch, and leaving the threshold at 0.2, `create org` finds nothing.

Reply:
Interesting! I wonder why that is... I'm not opposed to enabling extended search anyway though

Reply:
I can't find a clear answer either.
It doesn't work as good in some cases too.
`create a` returns result with our current config, but returns nothing with extended search.

Reply:
I think I got it. Extended search treats each word as a separate token unless quoted. Then merges the results from each token using AND logic. In the case above, `a` is less then the minimum of 2 characters, and has no results. When the results from searching `a` (nothing) are AND'd with the results from `create`, nothing shows. 

Non-extended search treats the whole string as one token.